webpack/webpack.parts.js

```diff
@@ -125,8 +131,70 @@ const wds = {
       "Access-Control-Allow-Methods": "GET, POST, OPTIONS",
       "Access-Control-Allow-Origin": "*",
     },
+    after: async (app) => {
+      const storiesJson = await extractStoriesJson({ stories, configDir });
+
+      addSideloadingAPI(app, storiesJson);
+      setupFileWatcher(projectDir);
+    },
   },
-};
+});
+
+// TODO: Handle updates (i.e. if something in glob changes).
+// Is there a hook for wds/wps for that?
+function addSideloadingAPI(app, storiesJson, projectDir) {
+  app.get("/api/ping", (req, res) => res.send("pong"));
+  app.get("/api/stories", (req, res) => res.json(storiesJson));
```

Comment:
This should become `stories.json` so it's isomorphic with the static build (it will use the same resources).